helm/squidex7/templates/deployment-worker.yaml

```diff
@@ -21,13 +21,17 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          securityContext:
+            {{- toYaml .Values.containerSecurityContext | nindent 12 }}
           ports:
             - name: http
-              containerPort: 80
+              containerPort: 10080
```

Comment:
This would be a breaking change, right? So we need a way to configure this. The setting should then also be mapped to ASPNETCORE__URLS environment variable.

Reply:
I just reviewed the documentation for other deployment methods for this software and found that a few supporting changes may be required on the https://github.com/Squidex/squidex-hosting repository to update the docker-compose files to proxy 80 and 443 to their respective ports exposed by the Dockerfile.

For example, I would propose changing the [docker-compose.yaml](https://github.com/Squidex/squidex-hosting/blob/master/docker-compose/docker-compose.yml) from:
```
  squidex_proxy:
    image: squidex/caddy-proxy
    ports:
      - "80:80"
      - "443:443"
```

to:
```
  squidex_proxy:
    image: squidex/caddy-proxy
    ports:
      - "80:10080"
      - "443:10443"
```

Reply:
Again, this would be a result of the changes to the Dockerfile, not these helm values.